importer.go

```diff
@@ -54,7 +87,7 @@ func (i *Importer) Import(path string) (*types.Package, error) {
 // be 0; it is reserved for future use.
 // Two calls to ImportFrom with the same path and srcDir return
 // the same package.
-func (i *Importer) ImportFrom(path, srcDir string, mode types.ImportMode) (*types.Package, error) {
+func (i *Importer) ImportFrom(path, srcDir string, mode types.ImportMode, filters FileFilters) (*types.Package, error) {
```

Comment:
This needs to satisfy `types.ImporterFrom` interface https://golang.org/pkg/go/types/#ImporterFrom